spec/rails_helper.rb

```diff
@@ -109,4 +115,24 @@ def delete_document; end
   c.default_cassette_options = { record: :new_episodes, re_record_interval: 7.days }
   c.ignore_hosts URI.parse(Rails.application.config_for(:elasticsearch).fetch("host")).host
   c.ignore_hosts 'pub.orcid.org', 'sandbox.orcid.org'
+
+  # FIXME: temporary workaround for feature specs
+  c.allow_http_connections_when_no_cassette = true
```

Comment:
My understanding was that webmock will raise error on external request when no cassette is used (even with this option). However it does not seem to work, maybe because of vcr hooking into webmock?

I will revert this change and make vcr ignore localhost requests.